recipes/clickhouse-cityhash/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: clickhouse-cityhash
+  version: "1.0.2"
+
+source:
+  fn: clickhouse-cityhash-1.0.2.tar.gz
+  url: https://pypi.python.org/packages/fb/e3/282f812aeace32817a6b2b894ccdfbe31146364294668cc85e1ad87b6f3f/clickhouse-cityhash-1.0.2.tar.gz
```

Comment:
Consider using the canonical url shown in:
https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18